File: android/src/main/kotlin/com/devfatani/webvuw/WebVuwPlugin.kt
package com.devfatani.webvuw

import io.flutter.plugin.common.PluginRegistry.Registrar

class WebVuwPlugin {

    companion object {

        @JvmStatic
        fun registerWith(registrar: Registrar) {
            registrar
                    .platformViewRegistry()
                    .registerViewFactory(
                            "plugins.devfatani.com/pull_to_refresh_web_view", WebVuwFactory(registrar.messenger()));
        }
    }
}

File: ios/Classes/WebVuwPlugin.m
#import "WebVuwPlugin.h"
#import "pull_to_refresh_web_view/pull_to_refresh_web_view-Swift.h"

@implementation WebVuwPlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
 
    [registrar registerViewFactory: [[WebVuwFactory alloc] initWithMessenger:[registrar messenger]] withId:@"plugins.devfatani.com/pull_to_refresh_web_view"];
}
@end
